demonstrations/tutorial_vqe_bond_dissociation.py

```diff
@@ -0,0 +1,389 @@
+r"""
+Modelling chemical reactions on a quantum computer
+==================================================
+
+.. meta::
+    :property="og:description": Construct potential energy surfaces for chemical reactions
+    :property="og:image": https://pennylane.ai/qml/_images/reaction.png
+
+.. related::
+   tutorial_quantum_chemistry Quantum Chemistry with PennyLane
+   tutorial_vqe Variational Quantum Eigensolver
+
+*Author: PennyLane dev team. Posted: . Last updated: .*
+
+The term "chemical reaction" is another name for the transformation of molecules -- the breaking and 
+forming of bonds. They are characterized by an energy barrier that determines
+the likelihood that the reaction occurs. These energy landscapes formed by these barriers are the
+key to understanding how chemical reactions occur, at the deepest possible level.
+
+.. figure:: /demonstrations/vqe_bond_dissociation/reaction.png
+    :width: 50%
+    :align: center
+
+    An example chemical reaction
```

Comment:
```suggestion
    An example chemical reaction.
```
That's an understatement :laughing: